amlb/utils/process.py

```diff
@@ -252,7 +252,7 @@ def run_cmd(cmd, *args, **kwargs):
             params[k] = kwargs[kk]
             del kwargs[kk]
     cmd_args = as_cmd_args(*args, **kwargs)
-    full_cmd = flatten([cmd]) + cmd_args
+    full_cmd = [os.environ["SHELL"], "-e"] + flatten([cmd]) + cmd_args
```

Comment:
This feels stupid, but it's the only way I could find to make the `-e` setting be in effect by default for the invoked command (which we want for the setup).